addon/components/flash-message.js

```diff
@@ -59,6 +60,14 @@ export default Component.extend({
     }
   }),
 
+  // Observer used here for backwards compatibility with Ember 1.11/1.12, replace
+  // with on('didRender') once possible.
+  _startTimers: on('didInsertElement', observer('flash', function() {
```

Comment:
Agreed, definitely don't want to use them unless there is no other way.

Here's the specific example that required this:

```
{{#flash-message flash=flashMessages.arrangedQueue.firstObject}}
  ...
{{/flash-message}}
```